install.sh

```diff
@@ -169,6 +169,9 @@ BOOTSTRAP_PACKAGES='zstd crew_mvdir ruby git ca_certificates libyaml openssl'
 [[ "${ARCH}" == "i686" ]] && BOOTSTRAP_PACKAGES+=' zlibpkg gcc_lib'
 
 if [[ -n "${CHROMEOS_RELEASE_CHROME_MILESTONE}" ]]; then
+  sudo cp /lib64/libc.so* /usr/local/
+  sudo chown chronos /usr/local/lib64/libc.so*
+  sudo chmod 777 /usr/local/lib64/libc.so*
```

Comment:
This only works for x86_64 architecture and we need to abstract `/usr/local` with `$CREW_PREFIX`.  What issue are you experiencing to require this change?  Maybe an issue should be opened to further explain your installation trouble?

Reply:
the one that you just fixed